pip/rapids-dask-dependency/pyproject.toml

```diff
@@ -13,7 +13,7 @@ version = "24.02.00a0"
 description = "Dask and Distributed version pinning for RAPIDS"
 dependencies = [
     "dask @ git+https://github.com/dask/dask.git@main",
-    "distributed @ git+https://github.com/dask/distributed.git@main",
+    "distributed @ git+https://github.com/dask/distributed.git@7562f9c566978de4f3f5b73920a24ea1813d6e28",
```

Comment:
Technically I think we could go one commit later ([964abea0fee437463585ffdfa5a8862ec7431f18](https://github.com/dask/distributed/commit/964abea0fee437463585ffdfa5a8862ec7431f18)) but since this is just a temporary workaround pinning I'm fine merging this as is.

Reply:
Yeah there wasn't a Conda package for that commit and thought it was better to pick one where both Conda & Pip used the same "version" to avoid potential discrepancies between the two